modules/articles/tests/client/articles.client.controller.tests.js

```diff
@@ -1,14 +1,14 @@
-'use strict';
-
 (function () {
+  'use strict';
+
   // Articles Controller Spec
   describe('Articles Controller Tests', function () {
     // Initialize global variables
     var ArticlesController,
       scope,
+      rootScope,
```

Comment:
@codydaig did you submit a PR for this? @trainerbill this really isn't writing tests, it's just renaming the variables.